dev/ci/README.md

```diff
@@ -0,0 +1,118 @@
+Continuous Integration
+======================
```

Comment:
Maybe you'd like to add "for the Coq Proof Assistant" as to make the context clearer.